File: atr_pkg/src/ma_node.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>
#include <iostream>
using namespace std;
void chao_callback(std_msgs::String msg)
{
    // cout << msg.data << endl;
    ROS_INFO(msg.data.c_str());
}
void yao_callback(std_msgs::String msg)
{
    ROS_WARN(msg.data.c_str());
}
int main(int argc, char *argv[])
{
    setlocale(LC_ALL, "zh_CN.UTF-8");
    ros::init(argc, argv, "ma_node");
    ros::NodeHandle nh;
    ros::Subscriber sub = nh.subscribe("kuai_shang_che_kai_hei_qun", 10, chao_callback);
    ros::Subscriber sub2 = nh.subscribe("giegie_daiwo", 10, yao_callback);
    while (ros::ok())
    {
        ros::spinOnce();
        // ros::Rate loop_rate(10);
    }
}

File: atr_pkg/scripts/ma_node.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# vim:fenc=utf-8
import rospy
from std_msgs.msg import String
from qq_msgs.msg import Carry


def chao_callback(msg):
    rospy.logwarn(msg.grade)
    rospy.logwarn(str(msg.star)+" star")
    rospy.loginfo(msg.data)


def yao_callback(msg):
    rospy.loginfo(msg.data)


if __name__ == "__main__":
    rospy.init_node("ma_node")
    sub = rospy.Subscriber("kuai_shang_che_kai_hei_qun", Carry, chao_callback)
    sub2 = rospy.Subscriber("gie_gie_dai_wo", String, yao_callback)
    rospy.spin()
